### DISCUSSION TITLE
Version Differences

### DISCUSSION THREAD

**Discussion by pjockey** [riffnshred/nhl-led-scoreboard#245] [Announcements]
Yesterday there was a core 1.7.0 update announcement and then later in the day, there was an update announcement for 1.6.1.

Just wonder the difference?

**Comment by falkyre** [comment]
That was us (@riffnshred and I) testing our github action workflows.  There is no release 1.7.0, it was the github actions changing the version number incorrectly

**Reply by riffnshred** [reply]
Its was a glitch in the Matrix xD
